src/conduits/java/com/enderio/conduits/common/items/ConduitProbeItem.java

```diff
@@ -0,0 +1,169 @@
+package com.enderio.conduits.common.items;
+
+import com.enderio.api.conduit.ConduitType;
+import com.enderio.api.misc.ColorControl;
+import com.enderio.api.misc.RedstoneControl;
+import com.enderio.conduits.common.conduit.ConduitBundle;
+import com.enderio.conduits.common.conduit.block.ConduitBlockEntity;
+import com.enderio.conduits.common.conduit.connection.ConnectionState;
+import com.enderio.conduits.common.conduit.connection.DynamicConnectionState;
+import com.enderio.conduits.common.network.C2SSetConduitConnectionState;
+import com.enderio.conduits.common.util.InteractionUtil;
+import com.enderio.core.common.network.CoreNetwork;
+import com.enderio.core.common.util.TooltipUtil;
+import net.minecraft.ChatFormatting;
+import net.minecraft.core.Direction;
+import net.minecraft.nbt.CompoundTag;
+import net.minecraft.network.chat.Component;
+import net.minecraft.world.InteractionResult;
+import net.minecraft.world.item.Item;
+import net.minecraft.world.item.ItemStack;
+import net.minecraft.world.item.TooltipFlag;
+import net.minecraft.world.item.context.UseOnContext;
+import net.minecraft.world.level.Level;
+import net.minecraft.world.level.block.entity.BlockEntity;
+import net.minecraftforge.common.util.INBTSerializable;
+import org.apache.commons.lang3.StringUtils;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.List;
+import java.util.Objects;
+
+public class ConduitProbeItem extends Item implements INBTSerializable<CompoundTag> {
+    public static final String STATE_FIELD = "STATE";
+    public static final String IS_INSERT = "IS_INSERT";
+    public static final String IS_EXTRACT = "IS_EXTRACT";
+    public static final String INSERT_CHANNEL = "INSERT_CHANNEL";
+    public static final String EXTRACT_CHANNEL = "EXTRACT_CHANNEL";
+    public static final String REDSTONE_CONTROL = "REDSTONE_CONTROL";
+    public static final String REDSTONE_CHANNEL = "REDSTONE_CHANNEL";
+    
+    private State state;
```

Comment:
State's should be stored in item NBT (in 1.20) or components (1.21), otherwise this setting is global to all of this item in the game instance.

Reply:
ok, should be fixed now